mods/ts/rules/civilian-infantry.yaml

```diff
@@ -270,13 +272,16 @@ VISC_LRG:
 		HP: 500
 	Armament:
 		Weapon: SlimeAttack
+		FireDelay: 10
```

Comment:
I was able to track down the problem: the critters don't reveal shroud, so they are not able to see anything around them.  This only worked for you because you disabled fog and shroud in the lobby options!